tslint.json

```diff
@@ -0,0 +1,15 @@
+{
+    "extends": [
```

Comment:
We do not use tslint anymore @priscilawebdev

Reply:
I completely forgot, but then I remembered ... so I didn't finish